src/schemas/execute.yaml

```diff
@@ -0,0 +1,277 @@
+      $ref: '#/components/schemas/AccountStorage'
+AccountStorage:
+  title: Storage slots for an account
+  type: object
+  additionalProperties:
+    - $ref: '#/components/schemas/hash32'
+BlockOverride:
+  title: Context fields related to the block being executed
+  type: object
+  properties:
+    number:
+      title: Number
+      $ref: '#/components/schemas/uint256'
+    prevRandao:
+      title: Randomness beacon
+      $ref: '#/components/schemas/uint256'
+    time:
+      title: Time
+      $ref: '#/components/schemas/uint256'
+    gasLimit:
+      title: Gas limit
+      $ref: '#/components/schemas/uint64'
+    feeRecipient:
+      title: feeRecipient
+      $ref: '#/components/schemas/address'
+    baseFee:
+      title: Base fee
+      $ref: '#/components/schemas/uint256'
+MultiCallResult:
+  title: Full results of multi call
+  type: array
+  items:
+    $ref: '#/components/schemas/MultiCallBlockResult'
+MultiCallBlockResult:
+  title: Result of multicall block-level, with array of calls
+  type: object
+  properties:
+    number:
+      title: Number
+      $ref: '#/components/schemas/uint256'
+    hash:
+      title: block hash
+      $ref: '#/components/schemas/hash32'
+    timestamp:
+      title: Time
+      $ref: '#/components/schemas/uint256'
+    gasLimit:
+      title: Gas limit
+      $ref: '#/components/schemas/uint64'
+    gasUsed:
+      title: Gas used
+      $ref: '#/components/schemas/uint64'
+    feeRecipient:
+      title: feeRecipient
+      $ref: '#/components/schemas/address'
+    baseFeePerGas:
+      title: Base fee
+      $ref: '#/components/schemas/uint256'
+    calls:
+      $ref: '#/components/schemas/CallResults'
+CallResults:
+  title: Results of multi call within block
+  type: array
+  items:
+    anyOf:
+      - $ref: '#/components/schemas/CallResultFailure'
+      - $ref: '#/components/schemas/CallResultSuccess'
+      - $ref: '#/components/schemas/CallResultInvalid'
+CallResultFailure:
+  title: Result of call failure
+  type: object
+  required:
+    - status
+    - return
+    - error
+    - gasUsed
+  properties:
+    status:
+      title: Call Status Failure
+      type: string
+      pattern: ^0x0$
+    return:
+      title: Return data
+      $ref: '#/components/schemas/bytes'
+    gasUsed:
+      title: Return gasUsed
+      $ref: '#/components/schemas/uint'
+    error:
+      title: Error Object
+      type: object
+      properties:
+        code:
+          title: Error code
+          type: number
+          $ref: '#/components/schemas/int'
+        message:
+          title: Error Message (execution reverted, out of gas, etc)
+          type: string
+        data:
+          title: Reverted, with optional message
+          type: string
+CallResultSuccess:
+  title: Result of call success
+  type: object
+  required:
+    - status
+    - return
+    - gasUsed
+  properties:
+    status:
+      title: Call Status Success
+      type: string
+      pattern: ^0x1$
+    return:
```

Comment:
```suggestion
    returnValue:
```

Reply:
Think returnValue or returnData is better?

Reply:
`returnData` as it may contain multiple Solidity "values".  It is really just a byte array.